### DISCUSSION TITLE
Acknowledging Multiple Alerts At Once

### DISCUSSION THREAD

**Discussion by NathanBVF** [Security-Onion-Solutions/securityonion#3985] [Unsupported Versions]
Is there a way that I can acknowledge similar alerts at once? For instance, if I see a scan from the same IP 100+ times is there a way that I can acknowledge all 100+ of them?

**Comment by weslambert** [comment]
Try `Group By Source IP, Name`:

![image](https://user-images.githubusercontent.com/16829864/116265382-1283a280-a749-11eb-9d85-bbad3c76e20b.png)
